source/config-file.txt

```diff
@@ -0,0 +1,69 @@
+.. _config-file:
+
+==========================
+Compass Configuration File 
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can configure |compass| settings using a configuration file. Configure 
+settings with a configuration file to maintain consistent behavior between 
+|compass-short| sessions. 
+
+Use Cases
+---------
+
+Use the configuration file to specify |compass-short| settings and behaviors. 
+For example:
+
+- If your deployment has strict privacy requirements, you can configure 
+  |compass-short| to block outgoing network connections.
+
+- If you need to restrict what actions users can perform in |compass-short|, you 
+  can disable the embedded shell and make |compass-short| read-only.
+
+- If you have multiple team members using |compass-short|, you can use a common 
+  configuration file to ensure a consistent user experience.
+
+Configuration File Location
+---------------------------
+
+- For macOS and Linux, the configuration file is located at ``/etc/mongodb-compass.conf``.
+
+- For Windows, the configuration file is located at ``<path\to\compass>\..\mongodb-compass.cfg``.
+
+Behavior
+--------
+
+When you configure settings with a configuration file:
+
+- You can not modify setting set in the configuration file after startup.
+
+- You must create your own configuration file. |compass-short| does not provide 
```

Comment:
[question]

This reads like Compass won't run at all unless a user creates a config file, which I don't think is the case. Maybe we should omit the first sentence in this bullet?

Reply:
Done.